### DISCUSSION TITLE
Taro上没有options属性

### DISCUSSION THREAD

**Discussion by lqw1** [NervJS/taro#15100] [Q & A] [answered]
![image](https://github.com/NervJS/taro/assets/28446076/c241a86e-b193-41a0-b028-c2fd7d80ff77)

![image](https://github.com/NervJS/taro/assets/28446076/e76f6296-19cd-4622-9c2b-47ba6198f37c)
是我的版本原因吗

**Comment by ZakaryCode** [comment, marked as answer]
当前版本确实存在该问题，后续修复，当前可以自行从 @tarojs/runtime 引入 options
